common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -265,6 +267,6 @@ DATA_IFACE=enx00ee22aa28ef
 PRPLMESH_MODE="CA" # CA = Controller & Agent, A = Agent only, C = Controller only
 
 # Export MultiAP libs folder
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:@BEEROCKS_INSTALL_PATH@/lib
+export LD_LIBRARY_PATH=@BEEROCKS_INSTALL_PATH@/lib:$LD_LIBRARY_PATH:@EXTRA_LD_PATH@
```

Comment:
Where is `EXTRA_LD_PATH` defined? not in this commit.

Reply:
It's in the CMakeLists.txt and is part of this commit...